README.md

```diff
@@ -61,73 +58,72 @@ Grab your API Key from the [Twilio SendGrid UI](https://app.sendgrid.com/setting
 
 Setup Environment Variables using the UI:
+
 1. Press Win+R and run SystemPropertiesAdvanced
 2. Click on Environment Variables
 3. Click New in user variables section
-4. Type SENDGRID_API_KEY in the name. (Make sure this name matches the name of key in your code)
+4. Type SENDGRID_APIKEY in the name. (Make sure this name matches the name of key in your code)
 5. Type actual API Key in the value
 6. Restart the IDE and you're done!
 
 Setup Environment Variables using CMD:
+
 1. Run CMD as administrator
-2. setx SENDGRID_API_KEY "YOUR_API_KEY"
+2. setx SENDGRID_APIKEY "YOUR_API_KEY"
 
-Here are few example to get and set API Keys programatically:
+Here are few example to get and set API Keys programmatically:
 
-Get Environment Variable
 ```csharp
-var apiKey = Environment.GetEnvironmentVariable("SENDGRID_API_KEY");
-```
+# Get Environment Variable
+var apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY");
 
-Set Environment Variable
-```csharp
-var setKey = Environment.SetEnvironmentVariable("SENDGRID_API_KEY", "YOUR_API_KEY");
+# Set Environment Variable
+var setKey = Environment.SetEnvironmentVariable("SENDGRID_APIKEY", "YOUR_API_KEY");
 ```
 
 ## Install Package
 
 To use Twilio SendGrid in your C# project, you can either <a href="https://github.com/sendgrid/sendgrid-csharp.git">download the Twilio SendGrid C# .NET libraries directly from our Github repository</a> or if you have the NuGet package manager installed, you can grab them automatically:
 
-```
-PM> Install-Package SendGrid
+```sh
+dotnet add package SendGrid
+
+# use Twilio SendGrid with HttpClientFactory
+dotnet add package SendGrid.Extensions.DependencyInjection
 ```
 
 Once you have the Twilio SendGrid library installed, you can include calls to them in your code.
 For sample implementations, see the [.NET Core Example](https://github.com/sendgrid/sendgrid-csharp/tree/master/ExampleCoreProject) and the [.NET 4.5.2 Example](https://github.com/sendgrid/sendgrid-csharp/tree/master/ExampleNet45Project) folders.
 
 ## Dependencies
 
-- Please see the [.csproj file](https://github.com/sendgrid/sendgrid-csharp/blob/master/src/SendGrid/SendGrid.csproj).
+Please see the [.csproj file](https://github.com/sendgrid/sendgrid-csharp/blob/master/src/SendGrid/SendGrid.csproj).
 
-<a name="quick-start"></a>
-# Quick Start
+# <a name="quick-start"></a>Quick Start
 
 ## Hello Email
 
 The following is the minimum needed code to send an simple email. Use this example, and modify the `apiKey`, `from` and `to` variables:
 
 ```csharp
-using SendGrid;
-using SendGrid.Helpers.Mail;
 using System;
 using System.Threading.Tasks;
+using SendGrid;
+using SendGrid.Helpers.Mail;
```

Comment:
nit: alphabetize

Reply:
It's fairly standard C# convention to put any custom namespaces after the BCL ones (`System.Foo`), this follows the conventions.

Reply:
Thank you for the clarification!